core/nodejsActionBase/src/service.js

```diff
@@ -58,11 +61,12 @@ function NodeActionService(config) {
      * @param app express app
      */
     this.start = function start(app) {
-        server = app.listen(app.get('port'), function() {
+        server = app.listen(config.port, function() {
```

Comment:
Priti I think this is the only relevant change in this file. Is that right?

Reply:
yup, this is the only logical change in addition to few format changes

Reply:
@rabbah it was our intent to use the config object...  good catch